pkg/telemetryquerylanguage/contexts/metrics/metrics_test.go

```diff
@@ -0,0 +1,1703 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tqlmetrics
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/telemetryquerylanguage/tql"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/telemetryquerylanguage/tql/tqltest"
+)
+
+func Test_newPathGetSetter_NumberDataPoint(t *testing.T) {
+	refNumberDataPoint := createNumberDataPointTelemetry(pmetric.NumberDataPointValueTypeInt)
+
+	newExemplars, newAttrs, newArrStr, newArrBool, newArrInt, newArrFloat, newArrBytes := createNewTelemetry()
+
+	tests := []struct {
+		name      string
+		path      []tql.Field
+		orig      interface{}
+		new       interface{}
```

Comment:
I know there's nothing technically wrong with using `new` as a field name (here and in other test files), but it's a bit jarring as a field name, and it confuses GitHub's syntax colorer. Would you consider renaming this field to, say, `newval` or even `newv`?

Reply:
No problem